starter-code/views/drones/index.ejs

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<body>
+<h2>Drones</h2>
+  <% drones.forEach((drone) => { %>
+    <ul>
```

Comment:
Not sure if this was the desired result but this makes every drone be a list instead of just a list item. If you wanted them each to just be a list item, then I'd move the `<ul>` tag before the forEach loop and the closing tag after the loop closes which it looks like you already have.